docs/_docs/thin-clients/java-thin-client.adoc

```diff
@@ -79,6 +79,10 @@ Note that the code above provides a failover mechanism in case of server node fa
 
 include::includes/partition-awareness.adoc[]
 
+Partition awareness functionality helps to avoid an additional network hop in the following scenarios:
+1. Single-key operations API, like #put, #get, etc. However, the functionality has no effect on those operations within explicit transactions (initiated via ClientTransaction.txStart() described in <<Executed transactions>> section).
+2. ScanQuery and IndexQuery accept a partition number as a parameter with which the query is routed to a particular server node that stores the requested data. Refer to <<Executing Scan Queries>> and <<Executing Index Queries>> sections for more information.
```

Comment:
Also, you should mark numerations correctly - to enforce every point starts  from new line. Currently, it is showed in a single line.